fixes scroll in yge pages

SaptakS opened the pull request:
<!--
  Thank you for your pull request. Please provide a description above and review
  the requirements below.

  Contributors guide: https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md
-->

##### Description
<!-- A description on what this PR aims to solve -->
Fixes scroll in yge pages.

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, ui, crypto, etc). -->

##### Testing
<!-- Why should the PR reviewer trust that this change doesn't break anything? How have you tested this change? -->

##### Refers/Fixes
<!--
  Link to an issue if applicable. For example:
  If your PR fixes an issue  -> Fixes: #102
  If your PR refers an issue -> Refs: #101
-->
Fixes #1616

owocki commented:
> Could we remove the quick-fix @owocki added ?

+1

thanks for the root cause fix yall

SaptakS commented:
I have removed the quick-fix and added the main fix [here](https://github.com/gitcoinco/web/pull/1666/commits/dc4831a0c389eb857210beff7bb6d0b4d961a1cc)
